src/Storages/System/StorageSystemZooKeeper.h

```diff
@@ -35,6 +35,7 @@ class StorageSystemZooKeeper final : public IStorage
     bool isSystemStorage() const override { return true; }
     bool supportsIndexForIn() const override { return true; }
     bool mayBenefitFromIndexForIn(const ASTPtr & node, ContextPtr, const StorageMetadataPtr &) const override;
+    bool parallelizeOutputAfterReading(ContextPtr) const override { return false; }
```

Comment:
I thought that maybe it could be enabled for `numbers_mt`, could be useful for tests for instance, but I'm ok with disabling it for all system tables.

Reply:
> I thought that maybe it could be enabled for numbers_mt

`numbers_mt` is not affected by this setting, since the pipeline is already paralleled by `number_mt` implementation. So, no resize is added [here](https://github.com/ClickHouse/ClickHouse/blob/22670aebf8fd54d2fc6f5c7cf64b8477729fcd02/src/Storages/IStorage.cpp#LL137-L141)